app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -121,6 +123,12 @@ public void NavigationTest() {
         }catch(Exception e){}
         Intents.intended(allOf(hasComponent(MainActivity.class.getName())));
 
+
+        Espresso.pressBack();
```

Comment:
I've seen this on a lot of apps, so I thought it would be nice. It keeps you from quitting unintentionally. Thanks to this, it also gives an example of the use of "Dialogs" and "onPressBack()".